assets/scripts/store/actions/ui.js

```diff
@@ -58,6 +59,18 @@ export function setDraggingType (draggingType) {
   }
 }
 
+/**
+ * Updates when a segment is being resized.
+ *
+ * @param {Boolean} isResizing - whether the resize action is being performed
+ */
+export function setSegmentIsResizing (isResizing) {
```

Comment:
I was wondering if `setSegmentIsResizing` is doing the same thing as `setDraggingType`? And if they are, should we continue using `setDraggingType` instead of `setSegmentIsResizing` or is having `setSegmentIsResizing` more understandable? 

`setDraggingType` can be 4 different values, one of which is [`DRAGGING_TYPE_RESIZE` ](https://github.com/streetmix/streetmix/pull/1197/files#diff-49915f825d9438f2ca07da7690ea06e4L41) so if we were to check if a segment is being dragged we can get it from `store.getState().ui.draggingType`.

Reply:
Hmm. The earlier PR this comes from (https://github.com/streetmix/streetmix/pull/1129/) removes the `DRAGGING_TYPE_*` stuff entirely. I think it still remains to be seen whether I keep going that direction or not.

OK, how about I sidestep this: instead of implying that `<ResizeGuides />`visibility is tied specifically to a "segment is resizing" state, we just instead call it "resize guides are visible" state. This might actually be better -- since we show guides the moment the user has pressed down on the resize handles, even if they haven't dragged or resized anything.

Reply:
👍 Sounds good!